features/support/ruby_features.rb

```diff
@@ -4,6 +4,6 @@
   if ::RSpec::Support::RubyFeatures.ripper_supported?
     block.call
   else
-    warn "Skipping scenario #{scenario.title} because Ripper is not supported"
```

Comment:
`title` [was removed in Cucumber 3.0](https://github.com/cucumber/cucumber-ruby/commit/def44923bff1baafdcc68fc20ce3e7c1ca565848#diff-e4760be9cc43f48844c83f32235d799f839409330648d85bcdad1a0b4109d75dL83).